tools/pubsys/src/aws/ami/wait.rs

```diff
@@ -11,8 +11,8 @@ use std::time::Duration;
 /// `success_required` checks in a row.
 pub(crate) async fn wait_for_ami(
     id: &str,
-    region: &Region,
-    sts_region: &Region,
+    region: &str,
```

Comment:
Is there no stronger type we can use than `&str`?... I'd be nervous about passing invalid regions around.

Reply:
Right now the region gets validated when the client configs are actually being generated (just before each client is created) so there shouldn't be a risk of a client getting an invalid region. I suspect the &str would be more efficient to pass around and then we don't have to convert it to a string at every error or debug message, but we could bring back and refactor the parse_region function if we want to make it a region object earlier.

The reason you're seeing these Region objects being replaced is they used to contain name and endpoint in a single object (because that's how rusoto handled endpoints) but now the endpoint is set separately and can just be fetched directly from pubsys-config's AwsConfig.